charts/kubermatic-operator/Chart.yaml

```diff
@@ -14,7 +14,7 @@
 
 apiVersion: v1
 name: kubermatic-operator
-version: 0.3.20
+version: 0.3.21
```

Comment:
This is already used tag, unfortunately

Reply:
I'm not sure why it's not yet detected as a git conflict, but it certainly will be.